src/handlers/createHandler.ts

```diff
@@ -162,6 +173,7 @@ export default function createHandler<
         }
         handlerIDToTag[props.id] = this.handlerTag;
       }
+      showRngh2NoticeIfNeeded();
```

Comment:
wrap this call with `if (__DEV__)` – this will allow this code block to be completely removed from release packages